src/test/EigenPod.t.sol

```diff
@@ -1498,4 +1497,18 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
             validatorInfo
         );
     }
+
+    function processPartialWithdrawal(
+        uint40 validatorIndex,
+        uint64 withdrawalHappenedTimestamp,
+        address recipient,
+        uint64 withdrawalAmountGwei
+    ) public returns(IEigenPod.VerifiedWithdrawal memory) {
+        return _processPartialWithdrawal(
+            validatorIndex,
+            withdrawalHappenedTimestamp,
+            recipient,
+            withdrawalAmountGwei
+        );
+    }
```

Comment:
I think this 'wrapper' contract (that you've named `EPInternalFunctions`) should probably be broken out into a separate file

Reply:
Fair, done